test-app/config/ember-try.js

```diff
@@ -56,38 +56,42 @@ module.exports = async function () {
         },
       },
       {
-        name: 'ember-release',
+        name: 'ember-4.0',
```

Comment:
I feel like I left a comment here this morning, but maybe it was resolved / deleted already?

How come we aren't trying 3.12 based on this line?: https://github.com/CrowdStrike/ember-browser-services/pull/193/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R28

Reply:
oh nvm I see the `lts-3.8` above now.